tests/test_sqs/test_sqs.py

```diff
@@ -777,6 +777,38 @@ def test_send_receive_message_with_attributes_with_labels():
     )
 
 
+@mock_sqs
+def test_change_message_visibility_than_permitted():
+
+    with freeze_time("2015-01-01 12:00:00"):
+        sqs = boto3.resource("sqs", region_name="us-east-1")
+        queue = sqs.create_queue(QueueName="blah")
+        queue.send_message(MessageBody="derp")
+
+        messages = queue.receive_messages()
+        messages.should.have.length_of(1)
+        messages[0].get("ReceiptHandle")
+
+        sqs.change_message_visibility(
+            queue_name="blah",
+            receipt_handle=messages[0].get("ReceiptHandle"),
+            visibility_timeout="360",
+        )
+
+    with freeze_time("2015-01-01 12:05:00"):
+
+        try:
```

Comment:
Please use the `with pytest.raises`-paradigm for exceptions like this
(https://github.com/spulec/moto/blob/master/tests/test_sqs/test_sqs.py#L241)

Reply:
Done